mozilla_django_oidc_db/schemas.py

```diff
@@ -0,0 +1,96 @@
+from django.utils.translation import gettext_lazy as _
+
+OPTIONS_SCHEMA = {
```

Comment:
What's the plan to map this schema to the `admin-oidc` config? Probably that identifier string should be reserved/defined in a constant in this library too so that it's not overwritten by downstream projects.

Reply:
yeah that was my idea - a dict/mapping of identifiers to options forms, and our migrations/apps.py ensures that those records exist at all times. This effectively means that we can probably also disable the "add config" button in the admin, since the known configs will be managed in code, upfront.